src/components/badges/style.js

```diff
@@ -21,7 +22,7 @@ export const Span = styled.span`
 `;
 
 export const ProBadge = styled(Span)`
-  background-color: ${props => props.theme.special.default};
+  background-color: ${theme.special.default};
   background-image: ${props =>
     Gradient(props.theme.special.alt, props.theme.special.default)};
```

Comment:
There's a bunch of cases my automatic move didn't catch, like this one—but it's a good enough start and didn't take a lot of work. 😅